loadrunner_professional/assets/logs/loadrunner_professional_tests.yaml

```
@@ -0,0 +1,100 @@
+id: "loadrunner_professional"
+tests:
+  - sample: >+
```

nit:

Do you mind using `|-` as a separator. It works the same but `|-` seems more commonly used for other integrations.
```suggestion
  - sample: |-
```